app/src/main/res/values/strings.xml

```diff
@@ -602,4 +602,10 @@ Upload your first media by tapping on the add button.</string>
   <string name="theme_default_name">Default</string>
   <string name="theme_dark_name">Dark</string>
   <string name="theme_light_name">Light</string>
+
+  <string name="cannot_open_location_settings">Failed to open location settings. Please turn on location and set it to High Accuracy</string>
+  <string name="set_location_mode_high_accuracy">Please make sure that location is set to both GPS and mobile networks</string>
```

Comment:
Just a few points about the strings:

* It would be great if we could make both consistent. We use "High Accuracy" in the failed message but explicitly mention the location providers in the other one.
* Are we not asking for more than what's required here? We just need the location turned on, right? So can't we just get away with showing the toast corresponding to 'set_location_mode_high_accuracy'? Or if we really need to show the toast it would be nice we re-phrase the string as: "Please turn on location. For best results, choose the High Accuracy mode."

Reply:
> * Are we not asking for more than what's required here? We just need the location turned on, right? So can't we just get away with showing the toast corresponding to 'set_location_mode_high_accuracy'?

If we show the toast corresponding to `set_location_mode_high_accuracy` and let's say that somehow the app failed to open the location settings then I am sure that it will definitely confuse the user if we just say to them `Please make sure that location is set to both GPS and mobile networks` because they just clicked on a button on a dialog that was gonna open the settings for them. But, that's just my view and I am open to ideas :)

>Or if we really need to show the toast it would be nice we re-phrase the string as: "Please turn on location. For best results, choose the High Accuracy mode."

This looks good and shouldn't we also add `Failed to open location settings` in its beginning?

Reply:
> If we show the toast corresponding to `set_location_mode_high_accuracy` and let's say that somehow the app failed to open the location settings then I am sure that it will definitely confuse the user if we just say to them Please make sure that location is set to both GPS and mobile networks because they just clicked on a button on a dialog that was gonna open the settings for them. But

Wait. That shouldn't happen, right? You show the `set_location_mode_high_accuracy` toast only when you are able to open the location settings and the `cannot_open_location_settings` toast when you aren't able to open the location settings. What am I missing? 🤔 

>> Or if we really need to show the toast it would be nice we re-phrase the string as: "Please turn on location. For best results, choose the High Accuracy mode."
> 
> This looks good and shouldn't we also add Failed to open location settings in its beginning?

Oh! Apologies for the confusion. That was an alternative suggestion for the `set_location_mode_high_accuracy` string.

Reply:
But you yourself said to me that a lot of devices do not have the exact High Accuracy mode.

Reply:
Or we could just say `For best results, choose the High Accuracy mode`.